static/src/js/components/CollectionSelector/__tests__/CollectionSelector.test.jsx

```diff
@@ -31,7 +31,8 @@ const setup = ({
           query: GET_PERMISSION_COLLECTIONS,
           variables: {
             params: {
-              limit: 100
+              limit: 100,
+              provider: null
```

Comment:
When the provider dropdown is in use, I think @trevorlang was aiming to use it as the default, essentially IF someone had gone through the effort to select one, use the hook to pull the previously selected value when we present it again, rather than wiping it out. So, we should be using the providerId from the hook rather than wiping it out by default.

Reply:
done